config/config.go

```diff
@@ -31,7 +31,7 @@ type Config struct {
 	}
 
 	Maintenance struct {
-		AlertCleanupDays int `public:"true" info:"Closed alerts will be deleted after this many days (0 means disable cleanup)."`
+		DataCleanupDays  int `public:"true" info:"Various data records including alerts and on-call history will be deleted after this many days (0 means disable cleanup)."`
```

Comment:
Can't change this since it's been in the wild and we need to preserve the intent, we'll need to add a new field and leave this one alone.